package.json

```diff
@@ -259,6 +259,17 @@
           },
           "markdownDescription": "Disable Qute validation for the given file name patterns.\n\nExample:\n```\n[\n \"**/*items.qute.*\"\n]```.",
           "scope": "resource"
+        },
+        "qute.validation.undefined.object": {
```

Comment:
I think the settings name should be:

`qute.validation.undefinedObject ` to map this settings with Java class `QuteValidationSettings#getUndefinedObject ()`